doc/guide/interact.rst

```diff
@@ -0,0 +1,354 @@
+
+       Print my most recently submitted jobid.
+
+       Show info about the specified job including why it ended.
+
+   * - ``flux cancel ƒuAsjAo``
+
+       ``flux kill -s HUP ƒuAsjAo``
+
+       ``flux pgrep -f pending .``
+
+       ``flux pkill sl..p``
+
+     - Cancel specified job.  See :man1:`flux-cancel`.
+
+       Send specified job a SIGHUP.  See :man1:`flux-job`.
+
+       List ids of all pending jobs.
+
+       Cancel all jobs named sleep or slurp.  See :man1:`flux-pgrep`.
+
+.. _connect_to_flux:
+
+Connecting to Flux
+------------------
+
+Flux commands need a Flux instance to talk to.  Which one?  Remember that batch
+jobs are Flux instances, allocations are Flux instances, and Slurm jobs can
+even be Flux instances.  Complicating matters, Flux instances can be launched
+recursively.
+
+local URI
+^^^^^^^^^
+
+Each instance, or more properly each Flux broker within an instance, can
+be contacted via a unique local URI.  The URI corresponds to a UNIX domain
+socket and looks something like::
+
+  local:///tmp/flux-lMDa6Z/local-0
+
+In the *initial program* (batch script, interactive alloc shell, or whatever),
+the FLUX_URI environment variable is set to the local URI of the rank 0
+broker.  Flux commands in the initial program, which also runs on rank 0,
+read FLUX_URI and reference the instance that started them.
+
+When running outside of an instance, FLUX_URI will not be set.  In this case,
+commands fall back to the compiled-in URI of the Flux system instance.
+When there isn't a broker of the system instance running on the local node,
+commands fail with an error like::
+
+  ERROR: Unable to connect to Flux: broker socket /run/flux/local was not found
+
+remote URI
+^^^^^^^^^^
+
+A Flux instance also has a remote URI that looks like::
+
+  ssh://test3/tmp/flux-lMDacZ/local-0
+
+This is the local URI above with the scheme changed to "ssh" and the hostname
+"test3" prepended to the path.  Given a job ID, :man1:`flux-uri` can look up
+the remote URI:
+
+.. code-block:: console
+
+  $ flux batch -N2 --wrap sleep 120
+  ƒcbUvuHDCiB
+  $ flux uri ƒcbUvuHDCiB
+  ssh://test3/tmp/flux-gMypIR/local-0
+  $
+
+Which can be used as follows:
+
+.. _sleep_example:
+
+.. code-block:: console
+
+  $ flux batch -N2 --wrap sleep 120
+  ƒcbUvuHDCiB
+  $ FLUX_URI=$(flux uri $(flux job last)) flux submit --cc 1-5 -N2 sleep 60
+  ƒ3croSDd
+  ƒ3ctHRVy
+  ƒ3ctHRVz
+  ƒ3cumQnK
+  ƒ3cwFQ4f
+  $ FLUX_URI=$(flux uri $(flux job last)) flux jobs
+       JOBID USER     NAME       ST NTASKS NNODES     TIME INFO
+    ƒ3ctHRVy alice    sleep       S      2      2        -
+    ƒ3ctHRVz alice    sleep       S      2      2        -
+    ƒ3cumQnK alice    sleep       S      2      2        -
+    ƒ3cwFQ4f alice    sleep       S      2      2        -
+    ƒ3croSDd alice    sleep       R      2      2   6.593s test[3-4]
+
+That started a batch job with a lifetime of 2m, then submitted 5 "sleep 60"
+jobs to it, then listed the batch job's active jobs.
```

Comment:
it took me awhile to figure out how it had a lifetime of 2m, i was looking for a `--time-limit`, perhaps just say "120 seconds".